R/scalar_in_linter.R

```diff
@@ -7,6 +7,8 @@
 #' `scalar %in% vector` is OK, because the alternative (`any(vector == scalar)`)
 #'   is more circuitous & potentially less clear.
 #'
+#' @param add_in_operators Character vector of additional functions that behave like the %in% operator
```

Comment:
not sure about the name, how about just `in_operators`?

Reply:
Shortend to `in_operators`